### PR TITLE
Updated version to 0.8.5 and added ping pong div views

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Review by ljmotta — APPROVED** [review]
LGTM! Nice PR @paulovmr !

File: packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx [hunk 1]
```diff
@@ -19,18 +19,24 @@ import { Envelope } from "@kogito-tooling/envelope";
...
 import { PingPongChannelApi, PingPongEnvelopeApi } from "../api";
-import { PingPongFactory } from "./PingPongFactory";
-import { PingPongEnvelopeContext } from "./PingPongEnvelopeContext";
-import { PingPongEnvelopeView, PingPongEnvelopeViewApi } from "./PingPongEnvelopeView";
-import { PingPongEnvelopeApiImpl } from "./PingPongEnvelopeApiImpl";
+import { PingPongFactory } from "../envelope/PingPongFactory";
+import { PingPongEnvelopeContext } from "../envelope/PingPongEnvelopeContext";
+import { PingPongEnvelopeView, PingPongEnvelopeViewApi } from "../envelope/PingPongEnvelopeView";
+import { PingPongEnvelopeApiImpl } from "../envelope/PingPongEnvelopeApiImpl";
+import {EnvelopeDivConfig, EnvelopeIFrameConfig} from "@kogito-tooling/envelope";
```
Comment (tiagobento) [inline packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx]: Code format

File: packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx [hunk 2]
```diff
@@ -19,18 +19,24 @@ import { Envelope } from "@kogito-tooling/envelope";
...
 
-export function init(args: { container: HTMLElement; bus: EnvelopeBus; pingPongViewFactory: PingPongFactory }) {
+export function init(args: {
+  envelopeConfig: EnvelopeDivConfig | EnvelopeIFrameConfig;
```
Comment (tiagobento) [inline packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx]: Can you rename that to `config`?

**Review by tiagobento — APPROVED** [review]
Nice! LGTM. Minor comments inline.

File: packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx (hunk 1 above)
Comment (paulovmr) [reply packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx]: Done.

File: packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx (hunk 2 above)
Comment (paulovmr) [reply packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx]: Done.

Comment (paulovmr) [reply packages/ping-pong-view/src/envelope/PingPongEnvelope.tsx]: Done.

**Review by tiagobento — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]

**Pull request merged**